velox/exec/tests/RowContainerTest.cpp

```diff
@@ -466,8 +466,40 @@ class RowContainerTest : public exec::test::RowContainerTestBase {
     }
   }
 
+  template <typename T, bool mayHaveNulls>
```

Comment:
Nit : Use of "mayHaveNulls" sounds ambiguous. Could you use just `hasNulls` ?

Reply:
I used the same name as used for the `equals` function. The template doesn't indicate if the input vector has nulls or not - it actually always has nulls because I didn't want to generate two of them in the caller. Instead, it is passed to the equals function to expect nulls or not. Thus, if this is `false`, then null values must be removed from the input vector.

Perhaps it should be named `equalsCanHandleNulls` or something?